pylib/Utilities/Logger.py

```diff
@@ -133,8 +159,76 @@ def stage_end_print(self, stagename, log):
         log['time'] = (stage_end-self.stage_start[stagename]).total_seconds()
         log['time_start'] = self.stage_start[stagename]
         log['time_end'] = stage_end
+        if self.elk:
+            elklog = {}
+            elklog['location'] = log['location'] if 'location' in log else None
+            elklog['section'] = log['section'] if 'section' in log else None
+            elklog['status'] = log['status'] if 'status' in log else None
+            elklog['stdout'] = log['stdout'] if 'stdout' in log else None
+            elklog['elapsed'] = log['time'] if 'time' in log else None
+            elklog['starttime'] = str(log['time_start']) if 'time_start' in log else None
+            elklog['endtime'] = str(log['time_end']) if 'time_end' in log else None
+            elklog['params'] = {k:v for k,v in log['parameters']} if 'parameters' in log else None
+            elklog['other'] = {}
+            if 'compiler' in log:
+                elklog['other']['compiler'] = log['compiler']['compiler']
+                elklog['other']['compiler_status'] = log['compiler']['status']
+                elklog['other']['compiler_version'] = log['compiler']['version']
+            for k in log:
+                if k != 'options' and k != 'parameters' \
+                and k != 'time_start' and k != 'time_end' and k != 'time' \
+                and k not in elklog and k not in elklog['other']:
+                    elklog['other'][k] = str(log[k])
+            print('SECTION', json.dumps(elklog))
+        self.current_section = None
+
+#{'compiler': {'compiler': 'gnu', 'status': 0, 'version': 'gcc (GCC) 8.3.0'},
```

Comment:
Did you intend to add these commented out fields

Reply:
nope, those were for reference. I'll delete them